src/boot/efi/devicetree.h

```diff
@@ -9,6 +9,26 @@ struct devicetree_state {
         void *orig;
 };
 
+#define FDT_BEGIN_NODE  0x00000001U
+#define FDT_END_NODE    0x00000002U
+#define FDT_PROP        0x00000003U
+#define FDT_NOP         0x00000004U
+#define FDT_END         0x00000009U
```

Comment:
Please use an enum for this.
```C
enum {
  FDT_BEGIN_NODE = 1,
  ...
  FDT_END        = 9,
}
```

(The type specifications are ignored. The compiler will select a type that allows enum values to fit, and it doesn't care if something like `U` or `C_INT64()` is used.)